Test/TemplateSourceTests.cs

```diff
@@ -2,22 +2,23 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Net.Http;
+using System.Text.Json;
```

Comment:
please explain why this is better than newtonsoft json

Reply:
The main reasoning was to cut external dependency, as it can easily be replaced with library which is part of .NET now.
https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/migrate-from-newtonsoft?pivots=dotnet-8-0

If you agree, maybe it's worth to move it into separate PR, to make the necessary changes both in test project and in template `KnownMimeTypes.tt`

Reply:
yes, please split it to a seperate PR